<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComPortMonitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Mode</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbHex">
         <property name="text">
          <string>Hex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbText">
         <property name="text">
          <string>Text</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnDisconnect">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Disconnect</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="gbTimer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Timer</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lTimer">
         <property name="text">
          <string>Период (мс)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spTimerPeriod">
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>500</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="leTimerMsg"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="btnTimer">
         <property name="text">
          <string>&amp;Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lSelectPort">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="5">
     <widget class="QGroupBox" name="gbMonitor">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Monitor</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0">
         <item>
          <widget class="QPushButton" name="btnMonitorClear">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSend">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="pteMonitor">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnConnect">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&amp;Connect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actConfigure"/>
   </widget>
   <widget class="QMenu" name="mode">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="actHex"/>
    <addaction name="actText"/>
   </widget>
   <addaction name="mode"/>
   <addaction name="tools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actConfigure">
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actHex">
   <property name="text">
    <string>hex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actText">
   <property name="text">
    <string>text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
